### ISSUE TITLE
Piped source with --no-check fails in v1.4.2

### ISSUE THREAD

**Issue by ericselin** [denoland/deno#7777]
Hi,

Starting with version 1.4.2, running a piped script fails when using `--no-check`.

```bash
> deno -V
deno 1.4.2
> echo "console.log('hello world')" | deno run -
Check file:///.../$deno$stdin.ts
hello world
> echo "console.log('hello world')" | deno run --no-check -
error: Cannot resolve module "file:///.../$deno$stdin.ts"
```

Piping with --no-check is useful e.g. for piping a bundled script via curl.

**Comment by bartlomieju** [comment]
Most likely the same problem as in https://github.com/denoland/deno/issues/7709; should be already fixed and released tomorrow.

**Comment by bartlomieju** [comment]
Fixed in #7748
